meteor/server/migration/X_X_X.ts

```diff
@@ -12,5 +13,26 @@ import { CURRENT_SYSTEM_VERSION } from './currentSystemVersion'
  */
 
 export const addSteps = addMigrationSteps(CURRENT_SYSTEM_VERSION, [
-	// Add some migrations!
+	{
+		id: 'Add missing ranks to ShowStyleVariants',
+		canBeRunAutomatically: true,
+		validate: () => {
+			return (
+				ShowStyleVariants.find({
+					_rank: { $exists: false },
+				}).count() > 0
+			)
+		},
+		migrate: () => {
+			ShowStyleVariants.find({
```

Comment:
It would have been nice with a comment here that we just assign any random rank for the variants, since they didn't have any previous order.